heartbeat/docs/heartbeat-options.asciidoc

```diff
@@ -478,6 +478,42 @@ check.response:
   body: '{"status": "ok"}'
 -------------------------------------------------------------------------------
 
+The following configuration shows how to check the response for multiple regex
+patterns:
+
+[source,yaml]
+-------------------------------------------------------------------------------
+- type: http
+  schedule: '@every 5s'
+  urls: ["https://myhost:80"]
+check.request:
```

Comment:
I think the indentation is off here. It should be on the same level as `urls` for example.

Same in all the examples below.

Reply:
Nice catch! The [current example](https://www.elastic.co/guide/en/beats/heartbeat/current/configuration-heartbeat-options.html) also has the wrong indentation too (which I copy pasted from)
![image](https://user-images.githubusercontent.com/4550136/37403898-8f68190c-2790-11e8-8227-7f30ea882281.png). So I'll fix up the original too.